src/tests/ftest/erasurecode/rebuild_disabled.yaml

```diff
@@ -20,7 +20,7 @@ hosts:
     - client-A
     - client-B
     - client-C
-timeout: 2500
+timeout: 3500
```

Comment:
Any idea why we are increasing the timeout in large magnitude? Is this ok?

Reply:
Yes, there is a performance issue and we still do not have Verbs in Ci so to get this test going in Ci I need to update the timeout.